src/screens/LoginPage/LoginPage.test.tsx

```diff
@@ -394,8 +394,6 @@ describe('Testing Login Page Screen', () => {
     expect(screen.getByTestId('passwordField')).toHaveFocus();
 
     expect(password.password.length).toBeLessThan(8);
-
-    expect(screen.queryByTestId('passwordCheck')).toBeInTheDocument();
```

Comment:
1. You have removed a test
2. You have not tested both valid and invalid variations of passwords.

Please be more professional in the submission of your code changes